.travis.yml

```diff
@@ -42,6 +42,9 @@ before_install:
     esac
 
     pip install pylint
+    pip install futures==3.1.1
+    pip install grpcio==1.4.0
```

Comment:
I'm not sure whether it makes sense to build grpc/grpcio from source here. It is significantly larger than the dependencies in python.bzl. grpc source is about 4.3 MB, as compared with the sizes of html5lib, six and werkzeug sources, which are all below 400 kB. grpc source contains C++ and Python, while the aforementioned smaller ones are mostly python. So building grpc from source would likely significantly increase the build time. In this sense,  grpcio is more similar to the depencencies: numpy and tensorflow, in that it should be directly installed from binary for the build environment. When the debugger plugin is ready for a public release, grpcio should be listed as a dependency of the tensorboard pip package (or at least stated explicitly as a requirement in the error message if it is not installed.)

I'd like to hear @jart's opinion.